### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by Farah189** [javiribera/locating-objects-without-bboxes#1]
Hi, Thanks for providing the code. It is mentioned in the code that a pre-trained model comes with this package. The pre-trained model "unet_256x256_sorghum.ckpt" is not available in checkpoints folder. I was trying to test the code initially on the provided test set. It is giving an error on this file while testing. Should I train it again with any data set first then?

**Comment by javiribera** [comment]
Hi @Farah189 , the code and instructions have substantially changed since your question. Can you check the checkpoints I uploaded to the new [readme](https://github.com/javiribera/locating-objects-without-bboxes/blob/master/README.md#pre-trained-models) file? Also the code now automatically detects if your dataset contains the ground truth in a CSV or an XML file, so you should be able to use the gt.csv

**Comment by Farah189** [comment]
Thanks for the response. I debugged all code to let it run on CPU using only CSV file. I am still learning Python so it took me a while to debug the code according to mine requirements. I trained it using my own dataset with 93% recall. 
One more thing that I wanted to ask: the network is trained with images of 256x256x3 dimension. The size of the test images should also be the same otherwise it gives an error in the last layer of the network. Can your modified code handle any dimension of the test images if it is trained with 256x256x3 training images?

**Comment by javiribera** [comment]
I'm going to close this issue since this new question has nothing to do with the title of the issue. Please open a new issue about it and refer to the commit number your are using. Please provide as many info as possible to help me reproduce the error, such as the input dataset (or a fraction of it), the terminal command, etc. Thank you,
